portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java

```diff
@@ -12,7 +12,17 @@
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello Waldemar Ramos Colon</h1>");
+    String json = "{"; 
```

Comment:
There is a much nicer way way to do this. Presumably you will do this in a subsequent PR / week when we discuss libraries?

Reply:
Sorry, But I don't understand what you are trying to say.

Reply:
A library like [gson](https://github.com/google/gson) would make it easier and less error prone to build JSON payloads. There is a week 3 lesson to use libraries. Converting this to `gson` then would be a nice improvement.

This comment requires no action on your part in this PR. It is a suggestion for future work.